CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 ## [1.8.12] - 2025-05-12
 [Full changelog](https://github.com/openfga/openfga/compare/v1.8.11...v1.8.12)
 
+### Added
+- Added pkg `migrate` to expose `.RunMigrations()` to be used for keeping consistent migrations running openFGA as a library
```

Comment:
This section needs to be moved up to Unreleased section, the text streamlined and link to PR added:
```suggestion
- Added pkg `migrate` to expose `.RunMigrations()` for programmatic use. [#2422](https://github.com/openfga/openfga/pull/2422)
```